tools/run_tests/performance/scenario_config.py

```diff
@@ -968,6 +968,90 @@ def __str__(self):
     return 'go'
 
 
+class RustLanguage:
```

Comment:
I think it has been added, otherwise benchmark can't be run.

Reply:
It has been deleted in a merge conflict.